In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

#4.	Using the pandas `read_csv()` method, read the GDP dataset into your notebook as a DataFrame called `gdp_df`. Take a look at the first few and last few rows to familiarize yourself with what is contained in this dataset.

In [4]:
gdp_df = pd.read_csv ("D:/Jai - NSS/python/projects/un-project-jaimreyes/data/gdp_percapita.csv")
gdp_df

Country or Area  Year        Value  Value Footnotes
0        Afghanistan  2023  1992.424394              NaN
1        Afghanistan  2022  1981.710168              NaN
2        Afghanistan  2021  2144.166570              NaN
3        Afghanistan  2020  2769.685745              NaN
4        Afghanistan  2019  2927.245144              NaN
...              ...   ...          ...              ...
8214        Zimbabwe  1994  3965.730986              NaN
8215        Zimbabwe  1993  3634.750494              NaN
8216        Zimbabwe  1992  3649.891947              NaN
8217        Zimbabwe  1991  4126.405247              NaN
8218        Zimbabwe  1990  4013.299059              NaN

[8219 rows x 4 columns]

#5. How many rows and columns are in `gdp_df`? What are the data types of each column? 
there are 8219 rows and 4 columns on in the df.

In [9]:
type(gdp_df)

pandas.core.frame.DataFrame

In [5]:
gdp_df = gdp_df.drop (columns = ['Value Footnotes'])
gdp_df

Country or Area  Year        Value
0        Afghanistan  2023  1992.424394
1        Afghanistan  2022  1981.710168
2        Afghanistan  2021  2144.166570
3        Afghanistan  2020  2769.685745
4        Afghanistan  2019  2927.245144
...              ...   ...          ...
8214        Zimbabwe  1994  3965.730986
8215        Zimbabwe  1993  3634.750494
8216        Zimbabwe  1992  3649.891947
8217        Zimbabwe  1991  4126.405247
8218        Zimbabwe  1990  4013.299059

[8219 rows x 3 columns]

In [8]:
gdp_df = gdp_df.rename (columns = {'Country or Area': 'Country', 'Value': 'GDP_Per_Capita'})
gdp_df

Country  Year  GDP_Per_Capita
0     Afghanistan  2023     1992.424394
1     Afghanistan  2022     1981.710168
2     Afghanistan  2021     2144.166570
3     Afghanistan  2020     2769.685745
4     Afghanistan  2019     2927.245144
...           ...   ...             ...
8214     Zimbabwe  1994     3965.730986
8215     Zimbabwe  1993     3634.750494
8216     Zimbabwe  1992     3649.891947
8217     Zimbabwe  1991     4126.405247
8218     Zimbabwe  1990     4013.299059

[8219 rows x 3 columns]